tools/cldr-apps/src/main/java/org/unicode/cldr/web/SearchManager.java

```diff
@@ -139,9 +170,67 @@ public void begin() {
             this.future = SurveyThreadManager.getExecutorService().submit(this);
         }
 
+        final CLDRLocale EN = CLDRLocale.getInstance("en");
+
         @Override
         public Search call() throws Exception {
             // first, simple search of exact values
+            addExactMatches(locale);
+
+            CLDRLocale cLoc = CLDRLocale.getInstance(locale);
+            CLDRLocale parLoc = cLoc.getParent();
+            while (response.isEmpty() && parLoc != null) {
+                addExactMatches(parLoc.getBaseName());
+                parLoc = parLoc.getParent();
+            }
```

Comment:
I don't think we want to search the parent, except for inherited values.

Reply:
This uses a non resolved file and searches the child first.